pepagent/pepagent.py

```diff
@@ -166,43 +209,45 @@ def get_project(
             _LOGGER.info("Files haven't been downloaded, returning empty project")
             return peppy.Project()
 
-        if found_prj is not None:
+        if found_prj:
             _LOGGER.info(f"Project has been found: {found_prj[0]}")
             project_value = found_prj[1]
             return peppy.Project(project_dict=project_value)
         else:
-            _LOGGER.warn(
+            _LOGGER.warning(
                 f"No project found for supplied input. Did you supply a valid namespace and project? {sql_q}"
             )
-            return None
+            return peppy.Project()
 
     def get_projects(
         self,
         registry_paths: Union[str, List[str]] = None,
```

Comment:
here you use `registry_paths`, which I think makes sense. in other places where you name the parameter just `registry`, I think is wrong.